Source/SpeechToTextV1/Models/Customization.swift

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright IBM Corporation 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import RestKit
+
+/** A custom language model. */
+public struct Customization: JSONDecodable {
+    
+    /// The ID of this custom language model.
+    public let customizationID: String
```

Comment:
Actually, if we return a `CustomizationID` struct instead of a `String`, users will now have to access the customization ID like so: `customization.customizationID.customizationID`.

Haha... what are your thoughts on this? I'm actually leaning towards keeping this property a `String`, because though it's a little less clear, it's a bit easier for the user to access the property.